src/lib/services/client-metrics/metrics-service-v2.test.ts

```diff
@@ -0,0 +1,117 @@
+import ClientMetricsServiceV2 from './metrics-service-v2';
+
+import getLogger from '../../../test/fixtures/no-logger';
+
+import createStores from '../../../test/fixtures/store';
+import EventEmitter from 'events';
+import { LastSeenService } from './last-seen-service';
+import { IUnleashConfig } from 'lib/types';
+
+function testSClientMetrics(flagEnabled = true) {
+    const stores = createStores();
+
+    const eventBus = new EventEmitter();
+    eventBus.emit = jest.fn();
+
+    // @ts-ignore only add config we care about
```

Comment:
do we need to use type assertion (as) and ts-ignore at the same time. Type assertion should do the work. There's also 'as unknown as IUnleashConfig' pattern if something doesn't match.

Reply:
`as IUnleashConfig` didn't work, but `as unknown as IUnleashConfig` worked but I'm not sure which one I prefer. The comment seems to be more explicit